source/common/protobuf/utility.cc

```diff
@@ -40,7 +40,11 @@ Json::ObjectSharedPtr WktUtil::getJsonObjectFromStruct(const Protobuf::Struct& m
   const auto status = Protobuf::util::MessageToJsonString(message, &json, json_options);
   // This should always succeed unless something crash-worthy such as out-of-memory.
   RELEASE_ASSERT(status.ok());
-  return Json::Factory::loadFromString(json);
+  return json;
+}
+
+Json::ObjectSharedPtr WktUtil::getJsonObjectFromStruct(const Protobuf::Struct& message) {
```

Comment:
Nit: since a `Protobuf::Struct` is a `Protobuf::Message`, this function should just be renamed `getJsonObjectFromMessage` (and corresponding arg change to `Message`, move it to `MessageUtil`).